.github/workflows/verify-pr-commit.yml

```diff
@@ -8,8 +8,13 @@ on:
     branches:
       - main
       - "*-development" # Feature Branches should suffix with -development
+# Set default permissions as restrictive
+permissions:
+  contents: read
+  packages: read
 env:
-  BIN_DIR: target/release
+  BIN_DIR_RELEASE: target/release
+  BIN_DIR_DEBUG: target/debug
```

Comment:
I changed some of the binaries built for the verify-pr workflow to be debug so they take less time.

Reply:
nit: Any way to reduce the duplication of value in the matrix?

Reply:
Managed to do this some.